### ISSUE TITLE
Prevent clearing of output other than the livelossplot graph

### ISSUE THREAD

**Issue by pinouchon** [stared/livelossplot#2]
I noticed that any output besides the loss graph of livelossplot gets cleared at each training iteration. (anytime I call `liveloss.draw()`)

Do you have a way of keeping that output ?
It seems that this mechanism is at the core of your package:

```python
# livelossplot/core.py:15
def draw_plot(logs, metrics, figsize=None, max_epoch=None,
              max_cols=2,
              validation_fmt="val_{}",
              metric2title={}):
    clear_output(wait=True) # I see what you did here
    plt.figure(figsize=figsize)
```

So I suspect that finding a workaround will be pretty difficult. With that said, I still would like to know if anyone has suggestions.

**Comment by stared** [comment]
@pinouchon I am aware of this "feature".  I mean, for Keras outputs it works for me - I can get a progress bar but avoid getting a wall of logs.

For other applications - well, I would love to learn if there is a workaround.

**Comment by stared** [comment]
The thread that motivated me to write `livelossplot` https://www.reddit.com/r/MachineLearning/comments/65jelb/d_live_loss_plots_inside_jupyter_notebook_for/ discusses various ways of clearing plots. Unless I miss something, I don't see a solution.

Some HTML-based techniques (with direct interactions with cell) may help - i.e. removing the previous version from HTML level.

**Comment by giovannipcarvalho** [comment]
There's a workaround: https://github.com/jupyter-widgets/ipywidgets/issues/1744

Basically have one cell with the following:
```
from ipywidgets import Output
OUTPUT_CONTEXT = Output()
display(OUTPUT_CONTEXT)
```

and wrap `liveloss.draw()` in the context:
```
[...]
with OUTPUT_CONTEXT:
    liveloss.draw()
[...]
```

The disadvantage is that the plots appear in a separate cell and above the training loop, but my tqdm bars are showing just fine (and not being cleared).

**Comment by stared** [comment]
With @Bartolo1024 we made a substantial rewrite, to allow custom operations before plotting, after each metrics subplot, and after all plots (starting from release `0.5.2`). See #73 and <https://github.com/stared/livelossplot/blob/master/examples/various_options.ipynb>.

Is it possible with the current setup, or some other changes are needed?
